components/Slider/src/MDCSlider.h

```diff
@@ -45,25 +45,128 @@ IB_DESIGNABLE
+ @c UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the
+ normal state.
+
+ @param fillColor The color of the filled track.
+ @param state The state of the slider.
+ */
+//- (void)setTrackFillColor:(nullable UIColor *)fillColor forState:(UIControlState)state;
+
+/**
+ Returns the track fill color associated with the specified state.
+
+ @params state The state that uses the track fill color.
+ @returns The track fill color for the specified state. If no color has been set for the specific
+          state, this method returns the color associated with the @c UIControlStateNormal state.
+ */
+//- (nullable UIColor *)trackFillColorForState:(UIControlState)state;
+
+/**
+ Sets the color of the ticks within the filled track to use for the specified state.
+
+ In general, if a property is not specified for a state, the default is to use the
+ @c UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the
+ normal state.
+
+ @param tickColor The color of the tick marks within the filled track.
+ @param state The state of the slider.
+ */
+//- (void)setFilledTrackTickColor:(nullable UIColor *)tickColor forState:(UIControlState)state;
+
+/**
+ Returns the tick color for the filled track portion associated with the specified state.
+
+ @params state The state that uses the filled-track tick color.
+ @returns The filled-track tick color for the specified state. If no color has been set for the
+          specific state, this method returns the color associated with the @c UIControlStateNormal
+          state.
+ */
+//- (nullable UIColor *)filledTrackTickColorForState:(UIControlState)state;
+
+/**
+ Sets the color of the ticks for the background (unfilled) track to use for the specified state.
+
+ In general, if a property is not specified for a state, the default is to use the
+ @c UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the
+ normal state.
+
+ @param tickColor The color of the tick marks outside the filled track.
+ @param state The state of the slider.
+ */
+//- (void)setBackgroundTrackTickColor:(nullable UIColor *)tickColor forState:(UIControlState)state;
+
+/**
+ Returns the tick color for the background (unfilled) track portion associated with the specified
+ state.
+
+ @params state The state that uses the background-track tick color.
+ @returns The background-track tick color for the specified state. If no color has been set for the
+          specific state, this method returns the color associated with the @c UIControlStateNormal
+          state.
+ */
+//- (nullable UIColor *)backgroundTrackTickColor:(UIControlState)state;
```

Comment:
Will these commented-out APIs be marked as deprecated?

Reply:
No, this is where I parked the API review. I'm going to be pushing up the actual changes in a series of PRs.